netstandard/ref/System.Runtime.InteropServices.ComTypes.cs

```diff
@@ -414,6 +466,8 @@ public partial interface ITypeInfo2 : System.Runtime.InteropServices.ComTypes.IT
         new void ReleaseTypeAttr(System.IntPtr pTypeAttr);
         new void ReleaseVarDesc(System.IntPtr pVarDesc);
     }
+    [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+    [System.Runtime.InteropServices.InterfaceTypeAttribute((System.Runtime.InteropServices.ComInterfaceType)(1))]
     public partial interface ITypeLib
```

Comment:
I don't understand why the actual enum name isn't being used? Why the enum value?